fints/segments/depot.py

```diff
@@ -0,0 +1,19 @@
+from . import FinTS3Segment
+
+class HKWPD(FinTS3Segment):
+    """
+    HKWPD (Depotaufstellung anfordern)
+    Section C.4.3.1
+    Example: HKWPD:3:7+23456::280:10020030+USD+2'
+    """
+    type = 'HKWPD'
+
+    def __init__(self, segno, version, account):
+        self.version = version
+        data = [
+            account
+            #'EUR'        # Währung der Depotaufstellung"
+            #2             # Kursqualität
```

Comment:
Well, I think this is just the reality of FinTS. The spec says that these fields are mandatory, but both 1822direkt and CortalConsors respond with an "invalid input" message if they are present. I added a comment below ``account`` explaining this.